cppSettings.gradle

```diff
@@ -180,7 +180,7 @@ subprojects {
                         // Gradle auto-adds the -m32 argument to the linker and compiler. Our compiler only supports
                         // arm, and doesn't understand this flag, so it is removed from both
                         cppCompiler.withArguments { args ->
-                            args << '-std=c++1y' << '-Wformat=2' << '-Wall' << '-Wextra' << '-Werror' << '-pedantic'
+                            args << '-std=c++11' << '-Wformat=2' << '-Wall' << '-Wextra' << '-Werror' << '-pedantic'
```

Comment:
Ah. On our other projects, Linux compiles with C++11, and athena compiles with C++1y. The new build system was running C++11 for both. We can always bump this later if we want, but I think for now knowing everything is full C++11 is good.